products/developments/python/geocode_dob.py

```diff
@@ -35,15 +29,15 @@ def geocode(input):
     except GeosupportError:
         try:
             geo = g["1B"](
-                street_name=sname,
-                house_number=hnum,
-                borough=borough,
+                street_name=input["address_street"],
+                house_number=input["address_numbr"],
+                borough=input["boro"],
                 mode="tpad",
             )
             geo = parse_output(geo)
             geo.update(
                 dict(
-                    uid=uid,
```

Comment:
Does changing these keys from uid to id change anything downstream?

Reply:
Yup - just used within the build as a unique row identifier for _init_devdb and downstream tables. They don't make it to export though. You'll see the same replacement in the sql files